mlflow/tracing/utils/__init__.py

```diff
@@ -234,6 +234,11 @@ def maybe_get_dependencies_schemas() -> Optional[dict]:
         return context.dependencies_schemas
 
 
+def maybe_get_model_id() -> Optional[str]:
```

Comment:
From the method name, it's unclear which model ID this gets. Can we rename the method to make that clearer and add a docstring?

Reply:
@dbczumar Added `logged_` and a docstring.